react/components/SearchResultContainer.js

```diff
@@ -105,6 +108,9 @@ export default class SearchResultContainer extends Component {
         },
       }
     )
+    // Need to redo this to get the rest and map with the pageProps
+    if (!!restParam) pageProps.query.rest = restParam
```

Comment:
`if (!!restParam) pageProps.query.rest = restParam` can be instead:
`restParam && pageProps.query.rest = restParam`.

Also applies for the next statement

Reply:
Can't do